bootloader/src/pyi_utils.c

```diff
@@ -298,30 +348,30 @@ pyi_get_temp_path(char *buffer, char *runtime_tmpdir)
             pyi_win32_utils_to_utf8(buffer, wchar_ret, PATH_MAX);
             free(wchar_ret);
             if (runtime_tmpdir != NULL) {
-              /*
-               * Restore TMP to what it was
-               */
-              if (original_tmpdir != NULL) {
-                pyi_setenv("TMP", original_tmpdir);
-                free(original_tmpdir);
-              } else {
-                pyi_unsetenv("TMP");
-              }
+                /*
+                * Restore TMP to what it was
+                */
+                if (original_tmpdir != NULL) {
+                    pyi_setenv("TMP", original_tmpdir);
+                    free(original_tmpdir);
+                } else {
+                    pyi_unsetenv("TMP");
+                }
```

Comment:
As much as I agree with what you've done here, no whitespace changes here please; it really messes with the blame.